client/lib/performance-tracking/components/performance-tracker-install.jsx

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable react/no-danger */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+
+const performanceTrackerInstall = ( { nonce } ) => {
```

Comment:
Typically we want react components to have PascalCased names
```suggestion
const PerformanceTrackerInstall = ( { nonce } ) => {
```

Reply:
Done